R/generate_dist.R

```diff
@@ -336,6 +345,34 @@
   new
 }
 
+# Internal function called by .generate - returns non-random data
+#
+# @param n The number of observations required in the data set
+# @param fn String that specifies the custom function
+# @param args String of comma delimited arguments that will be passed to fn
+# @param dtSim Incomplete simulated data.table
+# @return A data.frame column  with the updated simulated data
+
+.gencustom <- function(n, fn, args, dtSim, envir) {
+  
+  # new <- .evalWith(formula, .parseDotVars(formula, envir), dtSim, n)
+  
+  args <- gsub("\\s+", "", args) # remove any spaces
+  arg_l <- as.list(unlist(strsplit(args, ",")))
+  arg_l <- lapply(arg_l, function(a) as.list(unlist(strsplit(a, "="))))
```

Comment:
Ah, just one line below my question is answered :D so it either needs trimws here too or the gsub above.